app/controllers/votes_controller.rb

```diff
@@ -3,6 +3,7 @@ class VotesController < ApplicationController
 
   # POST /votes.json
   def create
+    # scope access würde die Überprüfung mit own? unnötig machen!
```

Comment:
I see why using a scope variable isn't required, but that doesn't make the user input any more trustable.